python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -1319,6 +1289,9 @@ def test_dataframe_error_context(self):
 
 
 class DataFrameTests(DataFrameTestsMixin, ReusedSQLTestCase):
+    def test_dataframe_error_context(self):
```

Comment:
why we need this test?

Reply:
Will remove. It was added for local testing purpose